packages/dds/tree/src/test/memory/tree.spec.ts

```diff
@@ -33,28 +39,65 @@ class RootNodeSchema extends builder.object("root-item", {
 	child: builder.optional(ChildNodeSchema),
 }) {}
 
-function createLocalSharedTree(id: string): TreeView<typeof RootNodeSchema> {
-	const sharedTree = SharedTree.create(
+const initialState = {
+	child: {
+		propertyOne: 128,
+		propertyTwo: {
+			itemOne: "",
+		},
+		propertyThree: new Map([["numberOne", 1]]),
+	},
+};
+
+// Array has 2 allowable types to make it less efficient for uniform chunks.
```

Comment:
Would be better to use proper doc comments here and below so the comment works with intelisense. It doesn't matter much in tests, but might as well use them when documenting a class.